config/profiles/mapping.json

```diff
@@ -25,6 +25,12 @@
   "mappings": {
     "gps": {
       "properties": {
+        "odsCode": {
```

Comment:
Shouldn't the idKey in config/esconfig.js also be odsCode? I guess both choicesId and odsCode are unique and indexable so perhaps not. Is it something you considered?

Reply:
Yes, at the moment odsCode isn't unique so we'd lose data if we used it as the idKey.
Currently we use choicesId as the idKey, as that was the unique index from the source data.
I'd prefer an ID field to be read explicitly by name, rather than implicitly by being the record ID for consistency. There are other ID lookups we may expose in future too, i.e. syndication ID.